frank-doc-doclet/src/main/java/org/frankframework/frankdoc/feature/Description.java

```diff
@@ -61,7 +61,7 @@ public String valueOf(FrankMethod method) {
 	public String valueOf(FrankClass clazz) {
 		String result = clazz.getJavaDoc();
 		try {
-			return Utils.replaceClassFieldValue(result, clazz);
+			return Utils.replaceClassCodeValue(Utils.replaceClassFieldValue(result, clazz));
```

Comment:
Ik zou graag willen dat deze wijziging wordt gecoverd met tests. De Frank!Doc is ingewikkeld, en de testen zijn een belangrijke bron van documentatie.